logstash-core/lib/logstash/api/modules/node.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+module LogStash
+  module Api
+    module Modules
+      class Node < ::LogStash::Api::Modules::Base
+        get "/" do
+          payload = {
```

Comment:
also, now that I think about it, this resource looks empty to me, why is like this?